src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        try {
+            return findByEmail(email).getPassword().equals(password);
```

Comment:
Potential NullPointerException. You are aware of findByEmail(email) returning null. Handle this case please